.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
If someone is using something other than vscode would this file still work?

Reply:
the .vscode/* do not do anything if not using vscode

Reply:
ok